source/speech/speech.py

```diff
@@ -745,7 +745,7 @@ def getObjectSpeech(
 	return sequence
 
 
-def _objectSpeech_calculateAllowedProps(reason, shouldReportTextContent):
+def _objectSpeech_calculateAllowedProps(reason, shouldReportTextContent, objRole):

```

Comment:
please add type hints here

Reply:
Done in acaeeee04805c58befa6dfded4877b892ce3929e.